### ISSUE TITLE
NeutralResourcesLanguageAttribute missing from Build.Utilities.Core

### ISSUE THREAD

**Issue by onovotny** [Microsoft/msbuild#2146]
Got the following warning with Update 3 preview trying to building a UWP test app that happens to pull this package in:

```
Severity	Code	Description	Project	File	Line	Suppression State
Warning		The assembly "C:\Users\oren\.nuget\packages\Microsoft.Build.Utilities.Core\15.1.548\lib\netstandard1.3\Microsoft.Build.Utilities.Core.dll" does not have a NeutralResourcesLanguageAttribute on it. To be used in an app package, portable libraries must define a NeutralResourcesLanguageAttribute on their main assembly (ie, the one containing code, not a satellite assembly).	System.Interactive.Tests.Uwp.DeviceRunner	C:\Users\oren\.nuget\packages\Microsoft.Build.Utilities.Core\15.1.548\lib\netstandard1.3\Microsoft.Build.Utilities.Core.dll		
```

**Comment by rainersigwald** [comment]
@cdmihai is this fixed by the addition of localization in #2106?

**Comment by cdmihai** [comment]
Just checked, this is fixed with the latest MSBuild packages: https://dotnet.myget.org/gallery/msbuild
